server/modules/activity/model/find.js

```diff
@@ -3,30 +3,29 @@ import { query } from '../../../database';
 const findSpendActivities = async (limit) => {
   const sql = `
                 SELECT
-                  sa.id "activity.id",
-                  sp.accessibility_website_url "activity.accessibility_link",
-                  sa.description "activity.description",
-                  sa.category "activity.category",
-                  sa.name "activity.name",
-                  sp.address_line1 "activity.address_line1",
-                  sp.address_line2 "activity.address_line2",
-                  sp.postcode "activity.postcode",
-                  sp.city "activity.city",
-                  sa.person_tc_price "activity.person_tc_price",
-                  sa.restrictions "activity.restrictions",
-                  sa.is_recurring "activity.is_recurring",
-                  -- a.start_date "activity.start_date",
-                  -- a.end_date "activity.end_date",
-                  i.file_name "activity.image_name",
-                  i.key "activity.image_key",
-                  i.bucket "activity.image_bucket",
-                  i.bucket_region "activity.image_bucket_region",
+                  sa.id "activity_id",
+                  sa.name "activity_name",
+                  sa.sub_title "subtitle",
+                  sa.person_tc_price "cost",
+                  sa.activity_types::VARCHAR[] "activity_types",
+                  sa.is_online "online",
+                  sp.city "townOrCity",
+                  i.file_name "image_name",
+                  i.key "image_key",
+                  i.bucket "image_bucket",
+                  i.bucket_region "image_bucket_region",
                   COALESCE(sp.name, eap.name) "spendVenue.name",
-                  COALESCE(sp.user_id, eap.user_id) "spendVenue.id"
+                  COALESCE(sp.user_id, eap.user_id) "spendVenue.id",
+                  (SELECT COUNT(sap.id)
+                     FROM spend_activity_participants sap
+                     WHERE sa.id = sap.spend_activity_id
+                     ORDER BY COUNT(sap.id) DESC
+                     ) AS "no_transactions"
                 FROM spend_activities sa
                 LEFT JOIN spend_venues_profiles sp ON (sa.user_id = sp.user_id)
                 LEFT JOIN earn_groups_profiles eap ON (sa.user_id = eap.user_id)
                 INNER JOIN images i ON (sa.image = i.id)
+                WHERE sa.is_live = 'true' AND sp.is_public = 'true'
```

Comment:
```suggestion
                WHERE sa.is_live IS true  AND (sp.is_public IS true OR eap.is_public IS true)
```

to get the spend activities been made by the earn groups